geodepy/tests/test_statistics.py

```diff
@@ -0,0 +1,150 @@
+import unittest
+from geodepy import statistics
+from geodepy.statistics import np
```

Comment:
Does this line import Numpy via the import statement in the statistics module? If it does, I didn't know you could do that! Is there an advantage doing it this way over just importing Numpy explicitly?

Reply:
When doing testing is better to do it like this than implicitly, because like this you are making sure in your tests that you are using the same import in your code and you are really testing properly